ui/ui-frontend/projects/archive-search/src/app/archive/validators/management-rules-validator.service.ts

```diff
@@ -76,9 +76,9 @@ export class ManagementRulesValidatorService {
       this.managementRules = data;
     });
 
-    if (this.managementRules.findIndex((managementRule) => managementRule.category === RuleTypeEnum.APPRAISALRULE) !== -1) {
+    if (this.managementRules.findIndex((managementRule) => managementRule.category === this.ruleCategorySelected) !== -1) {
```

Comment:
il faut aussi ajouter ça avant ta condition changée : 

`    this.managementRulesSharedDataService.getRuleCategory().subscribe((data) => {
      this.ruleCategorySelected = data;
    });`